src/test/java/org/assertj/core/internal/files/File_assertHasSize_Test.java

```diff
@@ -68,8 +69,9 @@ void should_fail_if_actual_is_not_a_file() {
     AssertionInfo info = someInfo();
     File notAFile = new File("xyz");
     // WHEN
-    expectAssertionError(() -> files.assertHasSizeInBytes(info, notAFile, 36L));
+    AssertionError error = expectAssertionError(() -> files.assertHasSizeInBytes(info, notAFile, 36L));
```

Comment:
`expectAssertionError` already makes sure the error is an `AssertionError`, please revert all the changes in this file.
The only thing to do is remove @DisplayName("Files.assertHasSizeInBytes:") as it is computed automatically

Reply:
this comment applies for the other tests too ;-)